Formula/ledit.rb

```diff
@@ -1,8 +1,9 @@
 class Ledit < Formula
   desc "Line editor for interactive commands"
   homepage "http://pauillac.inria.fr/~ddr/ledit/"
-  url "http://pauillac.inria.fr/~ddr/ledit/distrib/src/ledit-2.04.tgz"
-  sha256 "484897016b001b6d64eb24e38a213177b230ebf8e0d9808a969b62f012e70ee4"
+  url "https://github.com/chetmurthy/ledit/archive/ledit-2-05.tar.gz"
+  version "2.05"
```

Comment:
You shouldn't need to specify this, except when `brew` isn't parsing the version from the URL correctly.

Reply:
Yeah, w/o this it parses it as `2-05`:
```
; brew info ledit
ledit: stable 2-05 (bottled)
```